api/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
 
 # Create your models here.
+
+class Task (models.Model):
+
+	STATUS_CHOICES = (
+		('UNASSIGNED', 'Unassigned'),
+		('WORKINGON', 'Working On'),
+		('COMPLETED', 'Completed'),
+		)
```

Comment:
This looks very cool now! I like code that I can understand while I am reading it :)